aapcs32/aapcs32.rst

```diff
@@ -301,6 +301,9 @@ changes to the content of the document for that release.
   | 2021Q1| 12\ :sup:`th` April 2021            | Clarify what it means for a VFP CPRC argument to be correctly     |
   |       |                                     | aligned.                                                          |
   +-------+-------------------------------------+-------------------------------------------------------------------+
+  | 2023Q1| 5\ :sup:`th` February 2022          | Document mapping from C2x _BitInt(N) and unsigned _BitInt(N) types|
+  |       |                                     | to machine types.                                                 |
+  +-------+-------------------------------------+-------------------------------------------------------------------+
```

Comment:
On reviewing my own patch -- I guess it shouldn't include an update in this table since this is the table.
I'm now starting to think that maybe this table is just for changes in releases, and that should be made upon a release.
Would appreciate a confirmation/refutation on that from someone more in touch with the releases @stuij ?

Similar for the aapcs64.rst document.